TwoPage/src/androidTest/java/com/microsoft/device/display/samples/twopage/PageSwipeTest.kt

```diff
@@ -0,0 +1,185 @@
+     */
+    @Test
+    fun app_horizontalFold_pagesSwipeWithinLimits() {
+        composeTestRule.setContent {
+            TwoPageAppTheme {
+                TwoPageAppContent(viewWidth = 0, isDualScreen = false, foldSize = 0)
+            }
+        }
+
+        // Simulate horizontal fold
+        publisherRule.simulateHorizontalFold(composeTestRule)
+
+        swipeOnePageAtATime()
+    }
+
+    /**
+     * Swipes forward through all four pages and asserts that each page is visible at the correct time,
+     * then swipes backwards through all the pages again and asserts that each page is visible at the correct time
+     */
+    private fun swipeOnePageAtATime() {
+        val pageTags = listOf(R.string.page1_tag, R.string.page2_tag, R.string.page3_tag, R.string.page4_tag)
+
+        // Swipe forwards
+        for (page in 0..3) {
+            // Assert current page is visible
+            composeTestRule.onNodeWithTag(composeTestRule.getString(pageTags[page])).assertIsDisplayed()
+
+            // Swipe to next page
+            composeTestRule.onRoot().performGesture { swipeLeft() }
+
+            // Assert current page is no longer visible (unless on the last page)
+            when (page) {
+                3 ->
+                    composeTestRule.onNodeWithTag(composeTestRule.getString(pageTags[page])).assertIsDisplayed()
+                else ->
+                    composeTestRule.onNodeWithTag(composeTestRule.getString(pageTags[page])).assertIsNotDisplayed()
+            }
+        }
+
+        // Swipe backwards
+        for (page in 3 downTo 0) {
+            // Assert current page is visible
+            composeTestRule.onNodeWithTag(composeTestRule.getString(pageTags[page])).assertIsDisplayed()
+
+            // Swipe to previous page
+            composeTestRule.onRoot().performGesture { swipeRight() }
+
+            // Assert current page is no longer visible (unless on the first page)
+            when (page) {
+                0 ->
+                    composeTestRule.onNodeWithTag(composeTestRule.getString(pageTags[page])).assertIsDisplayed()
+                else ->
+                    composeTestRule.onNodeWithTag(composeTestRule.getString(pageTags[page])).assertIsNotDisplayed()
+            }
+        }
+    }
+
+    /**
+     * Tests that the three dual-screen page pairs ([1,2], [2,3], and [3,4]) only swipe between each other when
+     * a vertical fold is present
+     */
+    @Test
+    fun app_verticalFold_pagesSwipeWithinLimits() {
+        composeTestRule.setContent {
+            val configuration = LocalConfiguration.current
+            val windowMetrics = remember(configuration) {
+                WindowMetricsCalculator.getOrCreate()
+                    .computeCurrentWindowMetrics(composeTestRule.activity)
+            }
+            val windowWidthDp = with(LocalDensity.current) {
+                windowMetrics.bounds.width().toDp().value
```

Comment:
Can we use `LocalConfiguration.current.screenWidthDp` without `WindowMetricsCalculator` for the width of screen here?